### ISSUE TITLE
Add effects - Shadow

### ISSUE THREAD

**Issue by yakkowu** [Padax/w-components#135]
Defines what types of shadow effect are there in some suitable components.

**Comment by yakkowu** [comment]
# TASK COMPLETE!
<img src="https://user-images.githubusercontent.com/46119829/149636313-d74e8bad-4143-4ad4-91ef-85d6a9abadb4.png" width="80%">

At this time, I provide this three depths: "Near", "Middle", and "Far" from the background, which are named 10, 20, and 30 in sequence, hoping to provide everyone with a more intuitive choices.

⤷ Here's the currently file [in Figma](https://www.figma.com/file/5mAH9laNFI4q3oJeTN0sgq/Design-System-%5BWIP%5D?node-id=1351%3A387)

<img src="https://user-images.githubusercontent.com/46119829/149636488-397d2b2e-20d3-4a0b-84a2-f9324cf089be.png" width="80%">

For developers, every types of shadow are all including two values in CSS `box-shadow`.
In other words, each is composed of two shadows, you can click on of them in Figma and check the Inspect on the right side to learn more.

<img src="https://user-images.githubusercontent.com/46119829/149637354-504b77fe-001c-4714-b9b8-cfa23f0a6862.png" width="80%">

**Comment by nizniz187** [comment]
@yakkowu 
Is Dialog the only component which applies to this shadow effect for now?

**Comment by yakkowu** [comment]
> @yakkowu 
> Is Dialog the only component which applies to this shadow effect for now?

Yes, it's. Absolutely correct‼ ✨👍 thanks

**Comment by nizniz187** [comment]
Thanks for replying.
I'll close this issue now, and add a reference for the issue #25 .
